projects/opentrain/gtfs_pipeline/data_handler/DataHandler.java

```diff
@@ -56,7 +56,8 @@ private static boolean isServiceIdRelevantToday(CSVRecord record) {
     DateFormat dateFormat = new SimpleDateFormat("yyyyMMdd");
     int startDate = Integer.parseInt(record.get("start_date"));
     int endDate = Integer.parseInt(record.get("end_date"));
-    int currentDate = Integer.parseInt(dateFormat.format(new Date()));
+    //int currentDate = Integer.parseInt(dateFormat.format(new Date()));
+    int currentDate = 20190217;
```

Comment:
Please make this a `static final` constant.
```suggestion
    static final int FILTER_DATE = 20190217;
```